### PR TITLE
Update organiser page copy

### PR DESCRIPTION
**Pull request by richardwestenra**
- Add eligibility-check guidance
- Create 'Pairing' section and add pairing advice
- Add more information about dealing with problems as an organiser

### CODE REVIEW

File: _posts/2015-02-05-organiser-guide.md [hunk 1]
```diff
@@ -64,6 +69,10 @@ Encourage them to come to any social events following the workshop. Usually peop
...
+
+If someone comes to you with a complaint, act on it promptly. If attendees are involved, speak to them in a private area where you cannot be overheard, or send them an email after the event. When passing on negative feedback about a coach, you may need to ask the student’s permission first, as this information can be identifying.
+
+For difficult eligibility conversations, it often helps to remind people that our coaches, organisers and hosts donate their time and money to help those who have faced unfair barriers to entry into the tech world.
 
-If someone comes to you with a complaint, act on it promptly. If attendees are involved, speak to them or send them an email after the event. If you feel that you are unable to deal with this, you are more than welcome to send an email to <london@codebar.io> and we will do our best to help you out or deal with any issues.
+If you feel that you are unable to deal with this, you are more than welcome to send an email to <london@codebar.io> and we will do our best to help you out or deal with any issues.
```
Comment (KimberleyCook) [inline _posts/2015-02-05-organiser-guide.md]: could we change this email to hello@codebar.io, it wouldn't be down to the London team to sort this :)

**Review by KimberleyCook — CHANGES_REQUESTED** [review]
This is a great update, thanks.

Just one little amend though.

File: _posts/2015-02-05-organiser-guide.md (hunk 1 above)
Comment (richardwestenra) [reply _posts/2015-02-05-organiser-guide.md]: Not part of my PR, but sure thing.

**Comment by richardwestenra** [comment]
@KimberleyCook Done

**Review by KimberleyCook — APPROVED** [review]

**Pull request merged**